chair/role.html

```diff
@@ -97,6 +100,8 @@ <h3>What to look for when choosing a chair</h3>
 organization/management);</li>
 <li>Familiarity and/or willingness to learn W3C Process and
```

Comment:
s|and/or|or| - if you are familiar you don't need to "learn", and if you aren't then you do. (Although in any event we need people to ensure they follow any changes or updates).